src/components/Template.mdx

```diff
@@ -1,10 +1,10 @@
-import { Meta } from '@storybook/addon-docs';
+import { Meta } from '@storybook/blocks';
 
 <Meta title="Guide/Component Story MDX Template" />
 
 # Component Name Here
 
-**DATE ADDED**: [Date]
+**DATE ADDED**: \[Date]
```

Comment:
Is this `\` needed or a Typo?

Reply:
@Alderr i think Prettier/lint/storybook migration added things for no reason